src/actions/accounts.spec.js

```diff
@@ -1,11 +1,14 @@
 import { removeStats } from './accounts'
 import CozyClient from 'cozy-client'
 
-jest.mock('cozy-client')
-
-const client = new CozyClient()
+const client = new CozyClient({})
```

Comment:
not createMockClient() ?

Reply:
No

Reply:
This is not the point of the PR, PR that is already making things better by removing withMutations.

Reply:
J'avais pas vu la ligne suivante...